casper/src/main/scala/coop/rchain/casper/BlockStatus.scala

```diff
@@ -13,12 +13,11 @@ object BlockStatus {
   def invalidBlockNumber: BlockError       = InvalidBlock.InvalidBlockNumber
   def invalidRepeatDeploy: BlockError      = InvalidBlock.InvalidRepeatDeploy
   def invalidSequenceNumber: BlockError    = InvalidBlock.InvalidSequenceNumber
-  def invalidShardId: BlockError           = InvalidBlock.InvalidShardId
+  def invalidDeployShardId: BlockError     = InvalidBlock.InvalidDeployShardId
```

Comment:
Why its `invalidDeployShardId ` shardID still supposed to be related to a block? Or you want it per deploy?

Reply:
Yes, I just reuse it for slashable error if some deploy is with invalid shard ID.